### ISSUE TITLE
Undefined index: string

### ISSUE THREAD

**Issue by stonkeep** [Qoraiche/laravel-mail-editor#103]
**Describe the bug**
I'm getting the follow error after update to 2.0.6 version and PHP to 7.4.7

**To Reproduce**
I went to http://127.0.0.1:8000/maileclipse/mailables and get the follow error

**Laravel Application**: 5.8

**Maileclipse package version**: 2.0.6

**Screenshots**
![image](https://user-images.githubusercontent.com/3990028/88803718-48c4cf00-d183-11ea-9e2e-a3893dbf27db.png)

**Comment by ReeceM** [comment]
@stonkeep Thank you for reporting this, sorry for the delay in it being seen.

**Comment by jalallinux** [comment]
yes its bad bug.
you can uncomment string in TYPES.

`public const TYPES = [
        'int' => 31,
//        'string' => 'test_string', // not needed as it can be cast __toString()
        'bool' => false,
        'float' => 3.14159,
    ];
`

**Comment by jalallinux** [comment]
yes its bad bug. but you can uncomment string in TYPES.
```
public const TYPES = [
        'int' => 31,
//        'string' => 'test_string', // not needed as it can be cast __toString()
        'bool' => false,
        'float' => 3.14159,
    ];
```

**Comment by ReeceM** [comment]
@jalallinux That was left out because it overrides all the string output to any of the mailable objects

**Comment by ReeceM** [comment]
@jalallinux have you had this issue before too? Also if it is possible @stonkeep would you be able to provide an example of what your mailable looked like when the error occurred?

**Comment by ReeceM** [comment]
I think there is mainly an improvement for the checking in the logic on the types to return.

**Comment by jalallinux** [comment]
<img width="1082" alt="Screen Shot 1399-06-11 at 16 21 19" src="https://user-images.githubusercontent.com/37062636/91845935-4c121580-ec6f-11ea-99be-9fde68ea750d.png">

**Comment by jalallinux** [comment]
@ReeceM when i install package and publish vendor then open url(http://localhost:8000/maileclipse/mailables) this error has been occurred.

**Comment by ReeceM** [comment]
Okay, so it is on startup without any mailables created?

**Comment by jalallinux** [comment]
no. I already have 4 mailable class.

**Comment by jalallinux** [comment]
<img width="797" alt="Screen Shot 1399-06-11 at 16 49 00" src="https://user-images.githubusercontent.com/37062636/91850715-3b639e80-ec73-11ea-8dea-2bc8c45062fc.png">
@ReeceM i removed function input type (string) and its fixed.
thanks for your attention.

**Comment by ReeceM** [comment]
@jalallinux okay, thank you for providing info on the classes that you have.

I will have a look at the logic check for when to be using preset types. Thank you 👍

**Comment by ReeceM** [comment]
This should be fixed now with v2.2.1, the issue will be closed and if there is an issue again, please re-open it 👍 😃
